packages/block-editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -19,7 +19,7 @@ const FormatToolbar = ( { controls } ) => {
 					<Slot name={ `RichText.ToolbarControls.${ format }` } key={ format } />
 				) }
 				<Slot name="RichText.ToolbarControls">
-					{ ( fills ) => fills.length &&
+					{ ( fills ) => fills.length !== 0 &&
```

Comment:
This renders 0 if there's no fills.

Reply:
Now, I remember why I was using `isEmpty` in other places. I reviewed this code myself but I couldn't find a reason why it would be wrong 😅 

/cc @ellatrix

Reply:
> This renders 0 if there's no fills.

See also #14579